subsys/socket_offload/socket_offload.c

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I'm almost sure all these files should go into subsys/net/lib/sockets/.

Reply:
Yes.  This patch was quite removed from the networking stack, I didn't want to presume.  But, yes, ideally.